specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-08-01/workbooks_API.json

```diff
@@ -562,7 +565,7 @@
       "properties": {
         "kind": {
           "type": "string",
-          "description": "The kind of workbook. Choices are user and shared.",
+          "description": "The kind of workbook. Only valid value is shared.",
           "enum": [
```

Comment:
aside: in CDS do we validate /replace this with shared and/or ignore this now for the /workbooks paths?

Reply:
in CDS, we still let user set it, we should fix it to fail the request if the kind is not "shared"